static/js/components/LearnerSearch.js

```diff
@@ -220,7 +226,7 @@ export default class LearnerSearch extends SearchkitComponent {
             title="Final Grade in Selected Course"
```

Comment:
I decided to just go ahead with what you suggested anyway and explicitly remove all of the ones that shouldn't show up for learners because it's easier to test against.